internal/modulecontext/database.go

```diff
@@ -5,6 +5,9 @@ import (
 	"fmt"
 	"strconv"
 
+	"github.com/XSAM/otelsql"
+	semconv "go.opentelemetry.io/otel/semconv/v1.4.0"
```

Comment:
We have a test  in internal/observability/client_test.go to ensure the semconv version is consistent that you'll need to update to mention this file, _or_ alternatively change the wording to be more generalised.

(same)